packages/common/src/ticker.ts

```diff
@@ -59,13 +59,20 @@ export class Ticker {
     let ticker = document.createElement("div");
     ticker.classList.add("en__component");
     ticker.classList.add("en__component--ticker");
-    let str = '<div class="ticker">';
+    let str = `<div class="ticker">`;
     for (let i = 0; i < items.length; i++) {
       str += '<div class="ticker__item">' + items[i] + "</div>";
     }
     str = '<div id="engrid-ticker">' + str + "</div></div>";
     ticker.innerHTML = str;
     this.tickerElement?.parentElement?.insertBefore(ticker, this.tickerElement);
     this.tickerElement?.remove();
+    let tickerSelect = document.querySelector(".ticker");
+    console.log(tickerSelect);
+    let tickerWidth = (tickerSelect) ? getComputedStyle(tickerSelect).width : '1000';
+    tickerWidth = Math.round(parseInt(tickerWidth)).toString();
+    console.log(tickerWidth);
+    ticker.style.setProperty("--ticker-size", tickerWidth.toString());
```

Comment:
I think we can add the `--ticker-size` attribute BEFORE the `.ticker` gets created, maybe counting the chars inside the `items` loop? This way we don't need to query the DOM for the element we just added.

Reply:
I was able to reduce the tickerWidth selection to two lines. I originally was counting the characters in the `items` loop, but Bryan wanted me to use the width of the ticker instead of the character count because he thought it was a better option ([here's a link](https://app.productive.io/2650-4site-interactive-studios-inc/task/2488902?taskActivityId=36917050) to his comment on the choice). I could consult with him if you think character count is the better option though!

<img width="713" alt="Screen Shot 2022-04-19 at 6 24 30 PM" src="https://user-images.githubusercontent.com/57374659/164111496-64f58896-bebe-4454-9007-62d1b9d0327a.png">